lib/platform/vsts/index.js

```diff
@@ -0,0 +1,413 @@
+// @ts-check
+const vstsHelper = require('./vsts-helper');
+const gitApi = require('./vsts-got-wrapper');
+
+const config = {};
+
+module.exports = {
+  // Initialization
+  getRepos,
+  initRepo,
+  setBaseBranch,
+  // Search
+  getFileList,
+  // Branch
+  branchExists,
+  getAllRenovateBranches, // Not implemented yet
```

Comment:
This is OK, it just means Renovate won't clean up its branches automatically if you leave them open or reconfigure something.